zcash_primitives/src/jubjub/mod.rs

```diff
@@ -220,7 +220,7 @@ impl JubjubBls12 {
         {
             let mut pedersen_hash_generators = vec![];
 
-            for m in 0..5 {
+            for m in 0..6 {
```

Comment:
With #111 merged into master, this shouldn't be necessary.

Reply:
I checked during the rebase, and this is indeed necessary for the given test vectors.